.gitbook/contents/hc/4_deploy-smart-contract.md

```diff
@@ -0,0 +1,65 @@
+# Deploy the Smart Contract
+
+With a few adjustments to the `deploy.py` script, we can easily deploy our newly created smart contract.
```

Comment:
This page is out of date. The script is now named "deploy-local.py" and it calls other Foundry scripts in crates/types/contracts/hc_scripts/